OSX/build/macports_ansible/compileMythfrontendAnsible.zsh

```diff
@@ -68,6 +68,8 @@ if [ $OS_MAJOR -le 11 ] && [ $OS_MINOR -le 15 ]; then
   DATABASE_VERS=mariadb-10.5
 else
   DATABASE_VERS=mysql8
+# Comment out line above and uncomment the following to keep using mariadb
```

Comment:
I'm not sure that these comments are necessary.  

They certainly won't hurt, but the "official" mythtv supported database is mysql (I asked on the mailing list a while back).  As such, the default is correctly set in the script, and the user is given the ability to override as they see fit.

Another issue is mariadb-10.5 is very dated (but not end of life).  This should really be something like mariadb-10.11, but not all related other ports have been updated correctly (e.g. qt5-mysql-plugin)

Please remove the comments or provide an argument for keeping them.

Reply:
You are correct, I only had it in there for my own convenience when I was buildling. Yes the problem with upgrading database versions is the supporting plugins. I have removed the comments.